### PR TITLE
Add GCP Terraform config

### PR DESCRIPTION
**Pull request by ldx**
This is a simple and minimalistic TF config example, using GKE.

### CODE REVIEW

File: deploy/terraform-gcp/main.tf [hunk 1]
```diff
@@ -0,0 +1,168 @@
+provider "google" {
+  project = "excellent-ion-277200"
```
Comment (jrroman) [inline deploy/terraform-gcp/main.tf:2]: I believe the project will have to be filled in by the user or set by an env for their specific project in gcp.

File: deploy/terraform-gcp/main.tf [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    }
+
+    oauth_scopes = [
+      "https://www.googleapis.com/auth/logging.write",
+      "https://www.googleapis.com/auth/monitoring",
+    ]
+  }
+}
+
+resource "google_filestore_instance" "filestore" {
```
Comment (justnoise) [inline deploy/terraform-gcp/main.tf:71]: Do we want to include filestore along with this manifest?  I was under the impression that would be an enterprisey add-on.

File: deploy/terraform-gcp/main.tf (hunk 1 above)
Comment (ldx) [reply deploy/terraform-gcp/main.tf:2]: Thanks, that's a leftover from my test setup. I'll fix it.

File: deploy/terraform-gcp/main.tf (hunk 2 above)
Comment (ldx) [reply deploy/terraform-gcp/main.tf:71]: Yes, this is just a few lines, nothing fancy. The business logic for image caching will be in a separate controller and itzo add-on.

**Pull request merged**